arm/Microsoft.EventHub/namespaces/readme.md

```diff
@@ -43,9 +45,74 @@ This module deploys an EventHub namespace
 | `skuName` | string | `Standard` | `[Basic, Standard]` | Optional. event hub plan SKU name |
 | `tags` | object | `{object}` |  | Optional. Tags of the resource. |
 | `vNetId` | string |  |  | Optional. Virtual Network ID to lock down the Event Hub. |
-| `workspaceId` | string |  |  | Optional. Resource ID of log analytics. |
+| `workspaceId` | string |  |  | Optional. Resource identifier of log analytics. |
```

Comment:
Should not be rolled back. Please update in bicep file and here

Reply:
done